tracing-core/src/dispatcher.rs

```diff
@@ -315,13 +315,35 @@ impl Dispatch {
     /// This calls the [`drop_span`]  function on the [`Subscriber`] that this
     ///  `Dispatch` forwards to.
     ///
+    /// **Note:** the [`try_close`] function is functionally identical, but
```

Comment:
If these are functionally identical, should `drop_span` be marked as deprecated and removed in tracing 0.2?

Reply:
There's no reason it _can't_ be deprecated. I opted not to because it's fine to call `drop_span`, and most current callers of it don't care about the return value from `try_close`; it's only `Layered` that does, so I didn't see any reason we _have_ to get rid of it, beyond eventually making the API surface smaller. Happy to deprecate it if you think keeping the `Subscriber` API small is worth the warnings.

Reply:
Ah, gotcha. When making the "let's deprecate `drop_span`" suggestion, I had two assumptions in mind:

1. `tracing` already has some surface area in the API, and my stance on APIs is to keep the surface area as small as possible, perhaps more than is strictly necessary.
2. With the introduction of `Layered`, I expect that the primary consumer of `Subscriber`'s API will be `Layered`, and `Layered` alone. However, I can be convinced that this assumption would couple the two APIs too closely together.

Reply:
Yeah, as per our offline conversation, I agree that there's no real _reason_ to keep around both functions, and I'm fine with deprecating `drop_span`; it is also nice to have a smaller set of methods on `Subscriber`.